apps/site/test/site_web/controllers/schedule_controller_test.exs

```diff
@@ -23,27 +23,27 @@ defmodule SiteWeb.ScheduleControllerTest do
     end
 
     test "has the origin when it has been selected", %{conn: conn} do
-      conn = get(conn, trip_view_path(conn, :show, "1", origin: "2167", direction_id: "1"))
+      conn = get(conn, trip_view_path(conn, :show, "1", origin: "66", direction_id: "1"))
       html_response(conn, 200)
-      assert conn.assigns.origin.id == "2167"
+      assert conn.assigns.origin.id == "66"
     end
 
     test "finds a trip when origin has been selected", %{conn: conn} do
-      conn = get(conn, trip_view_path(conn, :show, "1", origin: "2167", direction_id: "1"))
+      conn = get(conn, trip_view_path(conn, :show, "1", origin: "66", direction_id: "1"))
       html_response(conn, 200)
-      assert conn.assigns.origin.id == "2167"
+      assert conn.assigns.origin.id == "66"
       assert conn.assigns.trip_info
     end
 
     test "finds a trip list with origin and destination", %{conn: conn} do
       conn =
         get(
           conn,
-          trip_view_path(conn, :show, "1", origin: "2167", destination: "82", direction_id: "1")
+          trip_view_path(conn, :show, "1", origin: "66", destination: "82", direction_id: "1")
         )
 
       html_response(conn, 200)
-      assert conn.assigns.origin.id == "2167"
+      assert conn.assigns.origin.id == "66"
```

Comment:
It'll be easier to update this in the future if you extract it to a `@modulevar`